.test-infra/metrics/docker-compose.yml

```diff
@@ -86,9 +86,35 @@ services:
       - DB_DBNAME=beam_metrics
       - DB_DBUSERNAME=admin
       - DB_DBPWD=<PGPasswordHere>
+  prometheus:
```

Comment:
I feel silly, but why do we need docker-compose configuration, and kubernetes configuration?

Reply:
We use docker-compose configuration to host the full stack on local machine. @Ardagan had provided docker-compose before (when introducing community metrics) and I decided not to drop support for it. This is beneficial for developers because they can use whatever tool they want for local development

Reply:
If we're able to provide Kubernetes only configuration files and use it in beam infra and for local development, then I'm in favor of dropping docker-compose. Keeping both tools can be confusing and can encourage future maintainers to use docker-compose not only for local dev (which IMO should be avoided).